CoEpi/ui/onboarding/OnboardingViewController.swift

```diff
@@ -100,6 +100,10 @@ class OnboardingViewController: UIViewController {
         joinButtonLabel.setTitle(L10n.Ux.Onboarding.join, for: .normal)
         questionsButtonLabel.setTitle(L10n.Ux.Onboarding.questions, for: .normal)
         
+        ButtonStyles.applyStyleUnselectedButton(to: joinButtonLabel)
```

Comment:
Let's name these methods just `applyUnselected` (`applyX(to:)` in general)?

Reply:
Done